frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -108,6 +114,18 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
         },
       };
 
+      if (pvcSC) {
+        restorePVCTemplate.spec.storageClassName = pvcSC;
+
+        // should set block only for RBD + RWX
+        if (
+          _.endsWith(storageProvisioner, cephRBDProvisionerSuffix) &&
+          accessModes[0] === 'ReadWriteMany'
+        ) {
+          restorePVCTemplate.spec.volumeMode = 'Block';
+        }
+      }
```

Comment:
Is it possible to have RBD + RWX of filesystem type?

Reply:
No not supported check https://github.com/ceph/ceph-csi#support-matrix

Reply:
UI doesn't allow us to create a PVC of RBD + RWX in Filesystem type, this combination will always result in creating a Block PVC from UI.

Reply:
@a2batic just trying to understand if there is any other scenario where block can be created from CLI & while restore we create filesystem-based.